server/src/main/java/com/defold/extender/services/HealthReporterService.java

```diff
@@ -64,13 +64,15 @@ public String collectHealthReport(boolean isRemoteBuildEnabled, Map<String, Remo
                 String instanceId = entry.getValue().getInstanceId();
                 // if instance controlled by instanceService and is currently suspended or suspending - mark it as 'operational'
                 if (instanceService != null && instanceService.isInstanceControlled(instanceId)) {
-                    if (instanceService.isInstanceSuspended(instanceId) || instanceService.isInstanceSuspending(instanceId)) {
+                    if (instanceService.isInstanceSuspended(instanceId) 
+                    || instanceService.isInstanceSuspending(instanceId)
+                    || instanceService.isInstanceStarting(instanceId)) {
                         updateOperationalStatus(platformOperationalStatus, platform, true);
                         continue;
+                    } else if (!instanceService.isInstanceRunning(instanceId)) {
+                        updateOperationalStatus(platformOperationalStatus, platform, false);
+                        continue;
                     }
-                } else if (!instanceService.isInstanceRunning(instanceId)) {
```

Comment:
Instance should check `isInstanceControlled`. Otherwise we will get error for instance that located outside of GCP.